crates/nu-source/src/meta.rs

```diff
@@ -362,6 +381,7 @@ impl Tag {
         Tag {
             span: Span::new(self.span.start, other.span.end),
             anchor: self.anchor.clone(),
+            data_color: None,
```

Comment:
Hmmmm, this actually might require more thought (this belongs to a function that merges 2 Tags)